static/app/components/onboarding/productSelection.tsx

```diff
@@ -16,11 +15,13 @@ import ConfigStore from 'sentry/stores/configStore';
 import {space} from 'sentry/styles/space';
 import type {Organization} from 'sentry/types/organization';
 import type {PlatformKey} from 'sentry/types/project';
+import {withChonk} from 'sentry/utils/theme/withChonk';
 import {useOnboardingQueryParams} from 'sentry/views/onboarding/components/useOnboardingQueryParams';
 
 interface DisabledProduct {
   reason: ReactNode;
   onClick?: () => void;
+  requiresUpgrade?: boolean;
```

Comment:
I have added this to be able to differentiate between a disabled state and one that requires a plan upgrade.